jujugui/static/gui/src/app/components/panel/_panel.scss

```diff
@@ -14,3 +13,16 @@
     left: $panel-space;
     width: $left-panel-width;
 }
+
+.panel-component.white-box {
+    position: absolute;
+    z-index: 1000;
+    top: $navbar-height;
+    bottom: $deployer-bar-height;
+    left: 0;
```

Comment:
I know these dimensions are just temporary but just so that it doesn't look like the env-size-display isn't shown could you add a few hundred to this. thx